api/client/service/update.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/swarm"
 	"github.com/docker/go-connections/nat"
+	shlex "github.com/flynn-archive/go-shlex"
```

Comment:
I can't find anything in the spec about it... but I'm pretty sure imports are split on `/` and `-`.